.github/workflows/wheels.yml

```diff
@@ -210,13 +211,46 @@ jobs:
         run: |
           CIBW="pip; args: --no-build-isolation"
           echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
-
+      
       - name: Build wheels
+        if: matrix.buildplat[1] != 'win' || matrix.buildplat[2] != 'ARM64' # win_arm64 build configured separately
```

Comment:
This doesn't actually need to be done separately, right? It's more maintainable to keep common things common. You can set the env vars you need in a step before the regular "Build wheels" step, and pass them on with:
```
echo "CC=clang-cl" >> $GITHUB_ENV
```

Reply:
Actually the syntax in gh-23133 looks even more concise, `$env:cc = "clang-cl"`

Reply:
Refactored that. Back to common cibuildwheel. Envs for Windows was combined, separate step for arm64 CIBW_BEFORE_BUILD_WINDOWS